app/src/pages/inside/testCaseLibraryPage/expandedOptions/expandedOptions.jsx

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2025 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import classNames from 'classnames/bind';
+import Parser from 'html-react-parser';
+import { useIntl } from 'react-intl';
+import { Button, BaseIconButton, SearchIcon, PlusIcon } from '@reportportal/ui-kit';
+
+import { EmptyStatePage } from 'pages/inside/common/emptyStatePage';
+import { ScrollWrapper } from 'components/main/scrollWrapper';
+import ImportIcon from 'common/img/import-thin-inline.svg';
+import PlusIconInline from 'common/img/plus-button-inline.svg';
+
+import { messages } from '../messages';
+import { FOLDERS } from './mockData';
+import { Folder } from './folder';
+
+import styles from './expandedOptions.scss';
+
+const cx = classNames.bind(styles);
+
+const defaultActiveFolder = 'All test cases';
+
+export const ExpandedOptions = () => {
+  const [activeFolder, setActiveFolder] = useState(defaultActiveFolder);
```

Comment:
I hope we can have kind of unique id for tree nodes. And we should store active path instead of name.
```
const [activePath, setActivePath] = useState(null);
```
`null` -> 'All test cases'
['id-root-5', 'id-node-3', 'id-leaf-7'] -> Active path in tree

Reply:
postponed